### PR TITLE
Issue serve static java script bundle instead of yarn dev server in dockerfile

### LINKED ISSUE #12: Serve static JavaScript bundle instead of yarn dev server in `Dockerfile`
The current frontend [Dockerfile](https://github.com/eclipse-tractusx/demand-capacity-mgmt/blob/consortia-cleanup/build/frontend/Dockerfile) is based on node and just runs the code with `yarn start` as dev server.
A better approach would be to build the app into static JavaScript bundles and place them in a [nginx-unprivileged](https://hub.docker.com/r/nginxinc/nginx-unprivileged) container

This is also conflicting with our Tractus-X release guidelines: [TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02)

### PR DESCRIPTION
**Pull request by Bailonis**
This PR serves to help keep our project aligned with the TRGs and other open-source policies.

In this case, we have build the app into static JavaScript bundles and place them in a nginx-unprivileged container

fix #12

### CODE REVIEW

File: build/frontend/Dockerfile [hunk 1]
```diff
@@ -18,25 +18,59 @@
...
+HEALTHCHECK NONE
+
+# add permissions for a user
+RUN chown -R 10000:3000 /app && chmod -R 775 /app/
+RUN chown 10000:3000 /entrypoint.sh && chmod -R 775 /entrypoint.sh
 
-# Install dependencies
-RUN yarn install
+# Install bash for env variables inject script
+RUN apk update && apk add --no-cache bash
```
Comment (SebastianBezold) [inline build/frontend/Dockerfile:67]: As stated in our release guidelines, We must not alter the base images of our published images.

File: build/frontend/NOTICE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Notice for Docker image
+
+DockerHub: [https://hub.docker.com/r/tractusx/demand-capacity-mgmt-frontend](https://hub.docker.com/r/tractusx/demand-capacity-mgmt-frontend)
+
+Eclipse Tractus-X product(s) installed within the image:
+
+__Demand Capacity Management Frontend__
+
+- GitHub: [https://github.com/eclipse-tractusx/demand-capacity-mgmt-frontend](https://github.com/eclipse-tractusx/demand-capacity-mgmt-frontend)
```
Comment (SebastianBezold) [inline build/frontend/NOTICE.md:9]: Wrong link

File: build/frontend/NOTICE.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- Project home: [https://projects.eclipse.org/projects/automotive.tractusx](https://projects.eclipse.org/projects/automotive.tractusx)
+- Dockerfile: [https://github.com/eclipse-tractusx/demand-capacity-mgmt-frontend/blob/main/Dockerfile](https://github.com/eclipse-tractusx/demand-capacity-mgmt-frontend/blob/main/Dockerfile)
```
Comment (SebastianBezold) [inline build/frontend/NOTICE.md:11]: Wrong link -> repo name does not match

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: build/frontend/NOTICE.md (hunk 2 above)
Comment (Bailonis) [reply build/frontend/NOTICE.md:9]: remove file

File: build/frontend/NOTICE.md (hunk 3 above)
Comment (Bailonis) [reply build/frontend/NOTICE.md:11]: file removed

**Comment by SebastianBezold** [comment]
Hi @Bailonis, as with your other PRs, this one also contains changes, that are not related to the issue that should be fixed.
Please isolate the changes

**Pull request closed without merging**